manifests/params.pp

```diff
@@ -16,8 +16,13 @@
         $service_file_template = 'confluence/confluence.initscript.erb'
         $service_lockfile      = '/var/lock/subsys/confluence'
         $refresh_systemd       = false
+      } elsif $facts['os']['name'] == 'Amazon' and $facts['os']['release']['major'] == '2018' {
+        $service_file_location = '/etc/init.d/confluence'
+        $service_file_template = 'confluence/confluence.initscript.erb'
+        $service_lockfile      = '/var/lock/subsys/confluence'
+        $refresh_systemd       = false
       } else {
-        fail("Only osfamily ${facts['os']['family']} 6 and 7 and supported")
+        fail("Only osfamily ${facts['os']['family']} 6 and 7, and Amazon Linux 2018 are supported")
```

Comment:
can you add the version to the metadata.json?

Reply:
I'll review. It looks like the logic has shifted to checking for systemd/sysv first. In that case this PR may be unnecessary. Let me get some testing and deeper dive.